src/features/device-state/state-patch-utils.ts

```diff
@@ -142,18 +144,15 @@ export type ImageInstallUpdateBody = {
 const shouldUpdateImageInstall = (() => {
 	const lastImageInstallReport = createMultiLevelStore<
 		ImageInstallUpdateBody & { updateTime: number }
```

Comment:
A comment that if the structure of the content changes, then we also need to mark it as `useVersion: true` would have been nice to have.